components/channel_info_modal/index.js

```diff
@@ -7,7 +7,7 @@ import {getCurrentRelativeTeamUrl, getCurrentTeam} from 'mattermost-redux/select
 
 import {getIsRhsOpen} from 'selectors/rhs';
 
-import ChannelInfoModal from './channel_info_modal.jsx';
+import ChannelInfoModal from './channel_info_modal.tsx';
```

Comment:
You can actually remove the file extension here entirely and it'll pick up the correct file for you.

Reply:
Good call, fixed.